web/src/views/SubmissionPortal/Components/SubmissionList.vue

```diff
@@ -14,14 +14,20 @@ import TitleBanner from '@/views/SubmissionPortal/Components/TitleBanner.vue';
 import IconBar from '@/views/SubmissionPortal/Components/IconBar.vue';
 import IntroBlurb from '@/views/SubmissionPortal/Components/IntroBlurb.vue';
 import ContactCard from '@/views/SubmissionPortal/Components/ContactCard.vue';
+import { SearchParams } from '@/data/api';
 import { deleteSubmission } from '../store/api';
-import { HARMONIZER_TEMPLATES, MetadataSubmissionRecord } from '@/views/SubmissionPortal/types';
+import { HARMONIZER_TEMPLATES, MetadataSubmissionRecord, PaginatedResponse } from '@/views/SubmissionPortal/types';
 
 const headers: DataTableHeader[] = [
   {
     text: 'Study Name',
     value: 'study_name',
   },
+  {
+    text: '',
+    value: 'is_test_submission',
+    sortable: false,
+  },
```

Comment:
I believe this needs to be removed still. Otherwise we get a column of the raw boolean values:
<img width="651" alt="image" src="https://github.com/user-attachments/assets/0f00afc7-866c-4c55-a6d5-5ab99f55ca75" />